modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -26,7 +26,11 @@ kubectl describe nodes
 
 == Limitations
 
-Redpanda does not support decreasing the CPU cores for brokers in an existing cluster.
+Redpanda supports decreasing the CPU cores for brokers in an existing cluster as of version 24.2.
+
+In v24.2, you must enable the `node_local_core_assignment` flag to be able to decrease the core count.
+
+In v24.3 or later, decreasing CPU cores is enabled by default.
```

Comment:
I think we need a separate PR for the beta version of 24.3 docs that removes the limitations section.

We don't expect users to switch off feature flags.